modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -531,6 +556,33 @@ public HttpResponse sendApprovalPostWithConsent(HttpClient client, String sessio
 		return sendPostRequestWithParameters(client, urlParameters, OAuth2Constant.APPROVAL_URL);
 	}
 
+	/**
+	 * Send approval post request for tenant with consent.
+	 *
+	 * @param client 				http client.
+	 * @param sessionDataKeyConsent session consent data.
+	 * @param consentClaims 		claims requiring user consent.
+	 * @param tenantDomain 			tenant domain.
+	 * @return http response.
+	 * @throws java.io.IOException IOException.
+	 */
+	public HttpResponse sendApprovalPostWithConsent(HttpClient client, String sessionDataKeyConsent,
+													List<NameValuePair> consentClaims, String tenantDomain)
+			throws IOException {
```

Comment:
Formating seems to be off here

Reply:
Seems to be due to git UI web view.

![Screenshot from 2023-09-27 12-49-27](https://github.com/wso2/product-is/assets/35653110/dd6cf7f6-5ea3-44dc-82d7-20b35a9b1724)